packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -272,7 +268,7 @@ export const SpacePlugin = (): PluginDefinition<SpacePluginProvides> => {
             const indices = getIndices(spaces.length);
             spaces.forEach((space, index) => {
               const update = () => {
-                const isDefaultSpace = defaultSpace && defaultSpace.key.equals(space.key);
+                const isDefaultSpace = client.spaces.default.key.equals(space.key);
```

Comment:
Feels like this could be rolled up into an API call b/c it will be used so often: `client.spaces.isDefaultSpace(space.key)`